.github/workflows/linting.yml

```diff
@@ -30,9 +31,9 @@ jobs:
             exit 1
           fi
 
-      - name: Run PHPCS
+      - name: Run Production
         run: |
-          OUTPUT=$(composer phpcs --no-check-publish 2>&1)
+          OUTPUT=$(composer phpcs-allow-todo --no-check-publish 2>&1)
```

Comment:
I think part of the problem here is that running commands through composer seems to hide the exit code. If I run `phpcs` directly, with errors I get exit code 2, with warnings I get exit code 1, and with no warnings/errors I get exit code 0. Running `composer phpcs` I always get exit code 0.